src/utils/pgwire/src/pg_extended.rs

```diff
@@ -35,31 +36,18 @@ use crate::types::Row;
 /// let params = parse_params(&type_description, &raw_params);
 /// assert_eq!(params, vec!["'A'", "'B'", "'C'"])
 /// ```
-fn parse_params(type_description: &[TypeOid], raw_params: &[Bytes]) -> Vec<String> {
+fn parse_params(type_description: &[TypeOid], raw_params: &[Bytes]) -> Result<Vec<String>, ()> {
```

Comment:
Can we return a variant of PsqlError here?

Reply:
We can replace it as PsqlError::BindError because it used in bind process.